xdsl/backend/csl/print_csl.py

```diff
@@ -72,6 +119,15 @@ def _var_use(
         else:
             return f"{intro} {self._get_variable_name_for(val)} : {self.mlir_type_to_csl_type(val.type)}"
 
+    def _export_sym_constness(self, ty: FunctionType | csl.PtrType):
+        if isinstance(ty, FunctionType):
+            return None
+        match ty.constness.data:
+            case csl.PtrConst.VAR:
+                return True
+            case csl.PtrConst.CONST:
+                return False
+
```

Comment:
I don't quite understand what this function does. Type annotations and docstrings would go a long way in making this more reviewable!

Reply:
When exporting symbols we have to specify if they are host-writable, except for functions, where it's a compile error to do so. 

Will add a docstring and return type